base/logger_audit.go

```diff
@@ -219,7 +219,7 @@ func (al *AuditLogger) shouldLog(id AuditID, ctx context.Context) bool {
 // shouldLogAuditEventForUserAndRole returns true if the request should be logged
 func shouldLogAuditEventForUserAndRole(logCtx *LogContext) bool {
 	if logCtx.UserDomain == "" && logCtx.Username == "" ||
```

Comment:
nit: I would prefer adding () around the && so that I don't have to worry about remembering operator precedence.